pkg/operator2/operator.go

```diff
@@ -83,9 +98,9 @@ func NewAuthenticationOperator(
 	coreInformers := kubeInformersNamespaced.Core().V1()
 	configV1Informers := configInformers.Config().V1()
 
-	authOpConfigNameFilter := operator.FilterByNames(authOperatorConfigResourceName)
-	osinNameFilter := operator.FilterByNames(targetName)
-	configNameFilter := operator.FilterByNames(configName)
+	authOpConfigNameFilter := operator.FilterByNames(authOpConfigResourceName)
```

Comment:
Nit: Since you're otherwise passing values from the `authOperator` struct, this should rather be `c.authOperatorConfigName`

Reply:
Fixed: 690bcaf9